spec/javascript/fixtures/application.js

```diff
@@ -13,8 +13,8 @@ const application = {
     "Id": "a0n0x000000AbE6AAK",
```

Comment:
These fixture updates are because I started going down the path of calling mapApplication on the SOQL app then checking that it matched the domain app exactly, but abandoned that for now in favor of explicitly testing the demographics changes.

I kept the changes that I made to fixtures that make addresses clearer and add a middle name